### PR TITLE
ENG 19130 use init --license or -l to stage license file before database starts

### PR DESCRIPTION
**Pull request by luyangco**
1. add a --license argument to the voltdb init command. The specified license file is copied into the database root directory.
2. Deprecate the use of --license on the voltdb start command.(For v10.0 release?)
3. Add the database root directory as the first location searched for the license at startup (but continue to look in the existing list of directories: working directory, home directory, and voltdb software root).

Unit tests in https://github.com/VoltDB/pro/pull/3043

### CODE REVIEW

File: src/frontend/org/voltdb/utils/MiscUtils.java [hunk 1]
```diff
@@ -306,30 +307,42 @@ public int getScheme()
      * For enterprise edition, look in default locations ./, ~/, jar file directory
```
Comment (JadenW) [inline src/frontend/org/voltdb/utils/MiscUtils.java]: nitpick: update the comment above

File: src/frontend/org/voltdb/RealVoltDB.java [hunk 2]
```diff
@@ -2575,6 +2580,30 @@ private void stageDeploymentFileForInitialize(Configuration config, DeploymentTy
...
+        File destF = new VoltFile(path, Constants.LICENSE_FILE_NAME);
+        if (config.m_forceVoltdbCreate && destF.exists()) {
+            destF.delete();
+        }
+        // copy new license to voltdb root
+        if (config.m_pathToLicense != null) {
+            File licenseF = new VoltFile(config.m_pathToLicense);
+            try {
+                Files.copy(licenseF, destF);
+                hostLog.info("Archive license file to VoltDB root directory: " + destF.getAbsolutePath());
```
Comment (dave-voltdb) [inline src/frontend/org/voltdb/RealVoltDB.java]: Nit: "Archive license file..." is an imperative, seems like you're telling someone to do something.  Maybe "License file archived to..." is a better way to report what has been done.

"Archived" seems a little odd to me, it's like you're putting a copy away that we will never use. Do we say "archive" elsewhere? If not, I suggest "copied".

Comment (luyangco) [reply src/frontend/org/voltdb/RealVoltDB.java]: Thanks, I'll follow your advice.

File: src/frontend/org/voltdb/utils/MiscUtils.java (hunk 1 above)
Comment (luyangco) [reply src/frontend/org/voltdb/utils/MiscUtils.java]: Will update the comment to reflect the new search path.

**Comment by dave-voltdb** [comment]
I suppose our kubernetes script (in tools/kubernetes) should be modified to use the new command, though this is not urgent as long as we still retain the 'old' way.  I can do that.

**Pull request merged**